content/concepts/feathersjs.textile

```diff
@@ -264,8 +264,12 @@ h2(#feathersjs-cost). Freedom - but at what cost?
+The problem with adding load balancers and multiple servers is the associated complexity it brings. If you’ve added on functions enough to operate a pub/sub service, maintaining a resilient pub/sub service takes time. Think "global redundancy":https://support.ably.io/support/solutions/folders/3000005442, "rate limiting":https://dzone.com/articles/building-a-distributed-rate-limiter-that-scales-ho, "history":/realtime/history and "message replay":https://support.ably.io/support/solutions/articles/3000044641, "message ordering":https://hackernoon.com/why-implementing-message-ordering-is-kind-of-a-big-deal-f2c0436e546c, "guaranteed delivery":https://support.ably.io/support/solutions/articles/3000044640, presence and "occupancy events":/tutorials/channel-occupancy-events, virtual "connection continuity":https://medium.com/ably-realtime/top-10-hardest-aspects-of-a-realtime-messaging-service-you-probably-didnt-know-about-but-matter-7edcebac6a5c - the list goes on. While FeathersJS makes it easy to get started with realtime, if your app starts to scale these additional layers to your stack will likely cause problems, with different aspects feeling somewhat disjointed when you start to encounter bursts in usage, users demanding idempotency, message ordering and so on.
 
 "Ably’s engineering blog":https://www.ably.io/blog/tag/ably-engineering/ describes problems and solutions associated with scaling realtime apps. When you find your (or your team’s) time taken up solving and/ or provisioning for these issues, that’s when - we believe - it’s worth looking for a cloud or better still serverless solution, leaving you free to concentrate on the core functionality of your app. That said, for small-scale applications or developers starting out with realtime - until realtime functionality scales and gets complicated, there is indeed a point at which FeathersJS could fit the bill of the holy grail of frameworks. Developers can judge for themselves when holy grail status starts to fade, and think about how to move on.
 
-*"Ably":https://www.ably.io/ is a global cloud network for streaming data and managing the full lifecycle of realtime APIs. To talk about scaling your app or service, get in touch with our dev-rel team.*
+"Ably":https://ably.io is a realtime messaging platform. We deliver billions of realtime messages every day to more than 50 million end-users across web, mobile, and IoT platforms.
+
+Developers use Ably’s 40+ SDKs to build realtime capabilities in their apps with our "multi-protocol pub/sub messaging":/channels, "presence":/documentation/core-features/presence, "push notifications":/push-notifications, "free streaming data sources":/hub from across industries like transport and finance, and "integrations that extend Ably":/reactor into third-party clouds and systems like AWS Kinesis and RabbitMQ.
```

Comment:
```suggestion
Developers use Ably’s 40+ SDKs to build realtime capabilities in their apps with our "multi-protocol pub/sub messaging":/channels, "presence":/core-features/presence, "push notifications":/push-notifications, "free streaming data sources":/hub from across industries like transport and finance, and "integrations that extend Ably":/reactor into third-party clouds and systems like AWS Kinesis and RabbitMQ.
```

Reply:
@MarkWoulfeAbly https://www.ably.io/core-features/presence will show me 404 page

Reply:
https://www.ably.io//documentation/core-features/presence will display the correct page, so your suggested code does not apply

Reply:
@Nosfheratu the link is actually broken as it is currently if you go to the page in the deployment and try and click on it. Every URL in the documentation repo will automatically generate the necessary /documentation part so you just need to use the part after, in this case, /core-features/presence.